components/PreScreeningMiddle/Footer.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <CommonCoolFooter v-if="buttons.length && !loading" height="fit-content">
+    <ElButton
+      v-for="button in buttons"
+      :key="button.label"
+      :type="button.type"
+      :disabled="button.disabled"
+      size="large"
+      @click="button.action"
+    >
+      {{ button.label }}
+    </ElButton>
+  </CommonCoolFooter>
+</template>
+
+<script setup lang="ts">
+import { isTodayAndPast } from "~/helpers/todayAndPast";
+
+const props = defineProps({
+  eventSlug: {
+    type: String,
+    required: true,
+  },
+});
+
+const eventStore = useEventStore();
+const userStore = useUserStore();
+const subscription = await userStore.getSubscription(props.eventSlug);
+const eventConfig = await eventStore.getEvent(props.eventSlug);
+
+interface Button {
+  label: string;
+  type: "primary" | "default";
+  disabled?: boolean;
+  visible: boolean;
+  action?: () => void;
+}
+
+const loading = ref(false);
+
+const isCanDonateOn = computed(() => {
+  const config = eventConfig?.preScreening;
+  return config && !config.disabled;
+});
+
+const isCanDonateMandatory = computed(() => {
+  const config = eventConfig?.preScreening;
+  return config && !config.disabled && config.mandatory;
+});
+
+const buttons = computed((): Button[] => {
+return [
+  {
+    label: "Pular questionário",
+    type: "default",
+    disabled: Boolean(isCanDonateMandatory.value),
+    visible: Boolean(isCanDonateOn.value),
```

Comment:
ai aqui ficaria:

```suggestion
    visible: Boolean(isCanDonateOn.value && !isCanDonateMandatory.value),
```

ou seja, visivel se estiver habilitado e não for obrigatório.